images/ceph/Makefile

```diff
@@ -20,24 +20,60 @@ include ../image.mk
 CEPH_VERSION = v14.2.2-20190722
 BASEIMAGE = ceph/ceph-$(GOARCH):$(CEPH_VERSION)
 CEPH_IMAGE = $(BUILD_REGISTRY)/ceph-$(GOARCH)
+OPERATOR_SDK_VERSION = v0.10.0
+GOHOST := GOOS=$(GOHOSTOS) GOARCH=$(GOHOSTARCH) go
 
 TEMP := $(shell mktemp -d)
 
+ifeq ($(HOST_PLATFORM),linux_amd64)
+OPERATOR_SDK_PLATFORM = x86_64-linux-gnu
+INCLUDE_CSV_TEMPLATES = true
+endif
+
+OPERATOR_SDK := $(TOOLS_HOST_DIR)/operator-sdk-$(OPERATOR_SDK_VERSION) 
+YQ := $(TOOLS_HOST_DIR)/yq.v2
+
 # ====================================================================================
 # Build Rook
 
-do.build:
+do.build: generate-csv-ceph-templates
 	@echo === docker build $(CEPH_IMAGE)
 	@cp Dockerfile $(TEMP)
 	@cp toolbox.sh $(TEMP)
 	@cp $(OUTPUT_DIR)/bin/linux_$(GOARCH)/rook $(TEMP)
 	@cp $(OUTPUT_DIR)/bin/linux_$(GOARCH)/rookflex $(TEMP)
 	@cp -r ../../cluster/examples/kubernetes/ceph/csi/template $(TEMP)/ceph-csi
 	@cp -r ../../cluster/examples/kubernetes/ceph/monitoring $(TEMP)/ceph-monitoring
+	# Copy temlated csv manifest only if operator-sdk is supported for the host platform.
```

Comment:
I see this line printed with every local build (typo: templated). The intention wasn't to print it, right?
```
=== docker build build-cf4cd2f5/ceph-amd64
# Copy temlated csv manifest only if operator-sdk is supported for the host platform.
sha256:b19a5c6e16acaaa8094886961eebecd45d593e89e3fb67d418b82d3512bd4d36
```

Reply:
fixed and updated pr.